components/nexus-security/src/main/java/org/sonatype/nexus/security/WebSecurityModule.java

```diff
@@ -56,7 +64,7 @@
  *
  * @since 2.6.1
  *
- * @see SecurityModule
+ * @see TestSecurityModule
```

Comment:
@mcculls I'm included to go with that for now ;-)  BTW glad you are removing it!